src/Surfnet/StepupMiddleware/ManagementBundle/Tests/Controller/InstitutionConfigurationControllerTest.php

```diff
@@ -66,6 +66,27 @@ public function authorization_is_required()
      * @test
      * @group management
      */
+    public function authorization_is_required_for_getting()
```

Comment:
`getting`->`querying_of_institution_configuration_options` `- Describe what the intended behaviour is, not the practical implementation (same with `posting` above).